updating/updating_a_cluster/updating-cluster-rhel-compute.adoc

```diff
@@ -12,49 +12,9 @@ WARNING: This assembly has been moved into a subdirectory for 4.14+. Changes to
 To do: Remove this comment once 4.13 docs are EOL.
 ////
 
-You can perform minor version and patch updates on an {product-title} cluster. If your cluster contains {op-system-base-full} machines, you must take additional steps to update those machines.
-
-[IMPORTANT]
-====
-The use of {op-system-base} compute machines on {product-title} clusters has been deprecated and will be removed in a future release.
-====
-
-== Prerequisites
-
-* Have access to the cluster as a user with `admin` privileges.
-See xref:../../authentication/using-rbac.adoc#using-rbac[Using RBAC to define and apply permissions].
-* Have a recent xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your update fails and you must restore your cluster to a previous state.
-* Your {op-system-base}7 workers are replaced with {op-system-base}8 or {op-system} workers. Red{nbsp}Hat does not support in-place {op-system-base}7 to {op-system-base}8 updates for {op-system-base} workers; those hosts must be replaced with a clean operating system install.
-* If your cluster uses manually maintained credentials, update the cloud provider resources for the new release. For more information, including how to determine if this is a requirement for your cluster, see xref:../../updating/preparing_for_updates/preparing-manual-creds-update.adoc#preparing-manual-creds-update[Preparing to update a cluster with manually maintained credentials].
-* If you run an Operator or you have configured any application with the pod disruption budget, you might experience an interruption during the update process. If `minAvailable` is set to 1 in `PodDisruptionBudget`, the nodes are drained to apply pending machine configs which might block the eviction process. If several nodes are rebooted, all the pods might run on only one node, and the `PodDisruptionBudget` field can prevent the node drain.
-
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../../architecture/architecture-installation.adoc#unmanaged-operators_architecture-installation[Support policy for unmanaged Operators]
-
-// Updating a cluster by using the web console
-include::modules/update-upgrading-web.adoc[leveloffset=+1]
+As of {product-title} 4.19, the use of {op-system-base-full} compute machines has been removed. You cannot perform updates on an {product-title} cluster if you have {op-system-base} machines. The {op-system-base} machines need to be removed before upgrading.
```

Comment:
I'd expect a release note saying package-managed RHEL is gone in 4.19.  And we already have [migration docs in 4.18][1] via @jldohmann in #91677.  It looks like we got release notes via #92899, and removed some other package-managed RHEL docs in .  And then these docs about working with your cluster would get completely removed, and were possibly just overlooked in the #92643 removals?

[1]: https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/updating_clusters/preparing-to-update-a-cluster#updating-cluster-prepare-past-4-18

Reply:
the file itself was untouched, but it was removed from the topic map

Reply:
Is there a reason to keep the file around once it's removed from the topic map?  It seems like removing it would reduce confusion like someone finding it while grepping and thinking that the text their grep hit was worth updating, without realizing that the text their grep hit in this file wasn't published anywhere.